apps/src/errorHandler.js

```diff
@@ -43,10 +40,10 @@ function outputError(warning, level, lineNum) {
   }
 
   // Send up to New Relic if it meets our sampling rate
-  if (level === ErrorLevel.ERROR && Math.random() < ERROR_LOG_RATE) {
+  if (level === ErrorLevel.ERROR) {
     logToCloud.addPageAction(logToCloud.PageAction.UserJavaScriptError, {
       error: warning
-    });
+    }, 1 / 20);
```

Comment:
Optional: it might be useful to strip the line numbers out of the JS error here, so that we can group by the error string to get a sense of the frequencies of each error type, unless it's easy to do that in a SQL/NRQL query somehow